README.md

```diff
@@ -9,6 +9,7 @@ A list of awesome Web3 apps, NFTs, libraries, tools and more.
   - [Git3](https://github.com/git3protocol/git3-cli) - A decentralized Git hosting protocol for Web3.
   - [BeFit](https://github.com/WilliamUW/ETHDenver2024) - BeFit revolutionizes fitness motivation by blending the spontaneity of BeReal with the accountability and rewards of exercising.
   - [Web3realtime Data](https://github.com/nishuzumi/web3realtime_data) - Web3 Realtime Data Website.
+  - [Hello Web3://](https://github.com/ethstorage/hello-web3) - Web3 Demo Website.
```

Comment:
The contract includes a FlatDirectoryFactory, which might be a bit overwhelming for users who are just starting with a “Hello World” example.